toolkit/scripts/srpm_pack.mk

```
@@ -40,6 +40,7 @@ $(call create_folder,$(BUILD_SRPMS_DIR))
 $(call create_folder,$(SRPM_BUILD_CHROOT_DIR))
 
 # General targets
+##help:target:input-srpms=Scan the local `*.spec` files, locate sources, and create `*.src.rpm` files.
```

```suggestion
##help:target:input-srpms=Scan the local `*.spec` files, locate sources, and create `*.src.rpm` files. Limit via SRPM_PACK_LIST.
```

This one is really powerful, good to call out how to use it I think.